modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -513,6 +513,12 @@ function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value
     $ss->assign("MOD", $mod_strings);
     $ss->assign("APP", $app_strings);
 
+    //In the case of a csv being generated check if the field type is a checkbox. if so just return the value
+    if($report_csv && $vardefFields[$fieldname]['type'] == 'bool'){
```

Comment:
1. Change "== 'bool'" to "=== 'bool'" to ensure they are of the same type.
2. PSR-2, need to have a space after the if and before the }.